thop/vision/basic_hooks.py

```diff
@@ -1,6 +1,6 @@
 import argparse
 import logging
-
+from .counter import *
```

Comment:
wildcard import should be avoided. Please explicity import the functions that you need.